<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form_createNewSwarmType1</class>
 <widget class="QDialog" name="form_createNewSwarmType1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>237</width>
    <height>141</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Input a label for new swarm </string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="baseSize">
        <size>
         <width>132</width>
         <height>291</height>
        </size>
       </property>
       <property name="title">
        <string>Create new swarm</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="sizeConstraint">
         <enum>QLayout::SetFixedSize</enum>
        </property>
        <item>
         <widget class="QLabel" name="label_localSwarmLabel">
          <property name="text">
           <string>Local swarm label:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="inputField_localSwarmLabel"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetMaximumSize</enum>
       </property>
       <item>
        <widget class="QPushButton" name="cmd_create">
         <property name="text">
          <string>Create a swarm</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cmd_cancel">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
